templates/CRM/Contribute/Form/ContributionPage/Tab.hlp

```diff
@@ -13,49 +13,49 @@
 {htxt id="id-configure-contrib-pages"}
 <table>
 <tr>
-    <td><a href="{crmURL p='civicrm/admin/contribute/settings' q="reset=1&action=update&id=`$contributionPageID`"}" id="idContribPageSettings"><i class="crm-i fa-chevron-right" aria-hidden="true"></i> {ts}Title and Settings{/ts}</a></td>
+    <td><a href="{crmURL p="civicrm/admin/contribute/settings" q="reset=1&action=update&id=`$contributionPageID`"}" id="idContribPageSettings"><i class="crm-i fa-chevron-right" aria-hidden="true"></i> {ts}Title and Settings{/ts}</a></td>
```

Comment:
I don't understand why Smarty complains about these single quotes in this hlp file, but not others.